.claude-plugin/README.md

```
@@ -0,0 +1,70 @@
+
+```bash
+# Add local marketplace
+/plugin marketplace add /path/to/unraid-mcp
+
+# Install the plugin
+/plugin install unraid @unraid-mcp
+```
+
+## Available Plugins
+
+### unraid
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)